File: C/src/1013.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int a,b,c;
    scanf("%i %i %i",&a,&b,&c);
    float maior = (a+b+abs(a-b))/2;
    int resultado = (c+maior+abs(c-maior))/2;
    printf("%i eh o maior\n",resultado);
    return 0;
}

File: go/1013.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c float64
	fmt.Scanln(&a, &b, &c)
	maior := (a + b + math.Abs(a-b)) / 2
	result := (maior + c + math.Abs(maior-c)) / 2
	fmt.Printf("%d eh o maior\n", int(result))
}
